### PR TITLE
Merge v2plugin & contiv install roles

### PR DESCRIPTION
**Pull request by neelimamukiri**

### CODE REVIEW

File: roles/contiv_network/defaults/main.yml [hunk 1]
```diff
@@ -18,22 +20,20 @@ ofnet_agent_port2: 9003
...
 bgp_port: 179
+# Since there is a conflict on vxlan port used by Docker overlay (ingress)
+# contiv needs to use a different port than the standard 4789 for v2plugin
+# vxlan_port: 8472
 vxlan_port: 4789
 netplugin_rule_comment: "contiv network traffic"
 
 aci_gw_image: "contiv/aci-gw"
+# contiv tar files to get netplugin binary
 contiv_network_version: "v0.1-11-30-2016.20-08-20.UTC"
```
Comment (rhim) [inline roles/contiv_network/defaults/main.yml]: What's the network_version variable on next line? Value looks old.

**Review by rhim — COMMENTED** [review]
LGTM. Minor issue with version string.

**Comment by neelimamukiri** [comment]
@rhim, PTAL. We usually don't update the version string here as it comes from contiv/install that we refer to.

**Review by gkvijay — APPROVED** [review]
LGTM. Thanks for taking care of this

**Pull request merged**